packages/react-label/src/components/Label/useLabelStyles.ts

```diff
@@ -1,8 +1,16 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
 import { tokens } from '@fluentui/react-theme';
-import type { LabelState } from './Label.types';
+import type { LabelSlots, LabelState } from './Label.types';
+import { SlotClassNames } from '@fluentui/react-utilities/src';
 
+/**
+ * @deprecated
+ */
 export const labelClassName = 'fui-Label';
```

Comment:
For `Label` you can actually remove this class name instead of deprecating it since the component is still under `unstable`:

```suggestion
```